src/content/workshops/functions/1-what-is-serverless/_index.md

```diff
@@ -4,12 +4,109 @@ weight: 101
 outputs: ["Reveal"]
 ---
 
-# Slide 1
+## What is Serverless?
 
-Test
+- Execution model
+- Pay as you consume
+- No need to control scaling
+- Event Driven
 
+{{% note %}}
+Serverless is an execution model. Everytime you hear the word serverless, it's talking about this cloud focused execution model. Running the same code locally is a different concept.
+
+An attribute of that execution model is that you pay for what you consume and not for what you reserve. When you create services in cloud providers, you reserve for a certain amount of CPU/RAM. This is not a necessary concern when you start a new serverless application.
```

Comment:
```suggestion
An attribute of the execution model is that you pay for what you consume rather than for reserved resources. When you create services in cloud providers, you reserve for your application a certain amount of CPU/RAM. This is not a necessary concern when you start a new serverless application.
```

When you say, 

> This is not a necessary concern when you start a new serverless application.

Are you trying to make the point that at the beginning of the project most devs aren't thinking about deployment, servers, and resources?

Reply:
Yes.